tmt/base.py

```diff
@@ -1539,6 +1541,40 @@ def lint_require_type_field(self) -> LinterReturn:
         yield LinterOutcome.FIXED, 'added type to requirements'
 
 
+@dataclasses.dataclass(repr=False)
+class LintableCollection(tmt.lint.Lintable['LintableCollection']):
+    """ Linting rules applied to a collection of Tests, Plans or Stories """
+
+    def __init__(self, objs: list["tmt.base.Core"], *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        self.objs = objs
+        self.cls_name = type(objs[0]).__name__ if len(objs) > 0 else ""
```

Comment:
Is this field used? I can find it neither in this patch nor in tmt code base.

Reply:
No, its not used anymore. It is a leftover from a previous refactoring that I missed to remove. 
Removed :heavy_check_mark: